DigitalLearningSolutions.Data/Services/CourseService.cs

```diff
@@ -24,5 +25,11 @@ public IEnumerable<CourseStatistics> GetTopCourseStatistics(int centreId, int ca
             var allCourses = courseDataService.GetCourseStatisticsAtCentreForCategoryId(centreId, categoryId);
             return allCourses.Where(c => c.Active).OrderByDescending(c => c.InProgressCount);
         }
+
+        public IEnumerable<CourseStatistics> GetCentreSpecificCourseStatistics(int centreId, int categoryId)
+        {
+            var allCourses = courseDataService.GetCourseStatisticsAtCentreForCategoryId(centreId, categoryId);
+            return allCourses.Where(c => c.CentreId == centreId && c.ArchivedDate == null);

```

Comment:
I've changed things around so the query itself only recovers non-archived courses. This might lead to some changes on the Top Courses page compared to the old site, as it doesn't look like the Top Courses query for the old site did not filter on ArchivedDate.